Framework/PythonInterface/test/python/mantid/api/MatrixWorkspaceTest.py

```diff
@@ -404,7 +404,7 @@ def test_setPlotType(self):
         self.assertEqual("plot", ws1.getPlotType())
 
         # test invalid doesn't take
-        ws1.setPlotType("invalid")
+        self.assertRaises(ValueError, ws1.setPlotType, "invalid")
```

Comment:
I'd recommend using `self.assertRaisesRegex` on these tests to make sure `setPlotType` is throwing the error you expect.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex